### ISSUE TITLE
Cannot compile on FreeBSD 13 with PostgreSQL 13 `lib/dllpg_query.so`

### ISSUE THREAD

**Issue by rawleyfowler** [roddyyaga/pg_query-ocaml#14]
Trying to compile on FreeBSD shows this:
```
File "test/dune", line 2, characters 8-12
2 |    (names test)
              ^^^^
Error: no rule found for lib/dllpg_query.so
```

**Comment by rawleyfowler** [comment]
You can get around this right now by just deleting the test folder. I will work on a fix.

**Comment by rawleyfowler** [comment]
This is fixed by #13
